packages/block-editor/README.md

```diff
@@ -12,6 +12,58 @@ npm install @wordpress/block-editor --save
 
 _This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats)._
 
+## Usage
+
+```js
+import {
+	BlockEditorProvider
+	BlockList
+} from '@wordpress/block-editor';
+import { useState } from '@wordpress/element';
+
+function MyEditorComponent () {
+	const [ blocks, updateBlocks ] = useState( [] );
+
+	return (
+		<BlockEditorProvider
```

Comment:
Any plans to introduce another wrapping component to make it as simple as:

```js
return (
	<BlockEditor
		value={ blocks }
		onInput={ updateBlocks }
		onChange={ updateBlocks }
	>
		<BlockList />
	</BlockEditor
);
```

or something like that.

Reply:
I ultimately think all these wrappers I'm using here should be part of `BlockEditorProvider` which is basically what you're proposing.

Reply:
Awesome, I like where it is heading 👍